util/parser.go

```diff
@@ -23,3 +24,20 @@ func ParseMemSize(memory string) (int64, error) {
 	}
 	return size, nil
 }
+
+func ParseEnvironmentList(envList []string) (map[string]string, error) {
+	res := make(map[string]string)
+
+	for _, part := range envList {
+		if keyValue := strings.SplitN(part, "=", 2); len(keyValue) < 2 {
+			return nil, fmt.Errorf("failed to parse --env argument '%s': missing =", part)
+		} else if strings.Contains(keyValue[0], " ") {
+			return nil, fmt.Errorf("failed to parse --env argument '%s': key must not contain spaces", part)
+		} else if strings.ContainsAny(keyValue[1], "= ") {
```

Comment:
Replace with `strings.Contains(keyValue[1], " ") {` to adhere the `putenv` format.